src/spikeinterface/core/datasets.py

```diff
@@ -1,14 +1,21 @@
 """
 Some simple function to retrieve public datasets with datalad
 """
+from __future__ import annotations
 
 from pathlib import Path
 
 import warnings
-from .globals import get_global_dataset_folder, is_set_global_dataset_folder
+from .globals import get_global_dataset_folder
 
```

Comment:
```suggestion
DEFAULT_REPO = "gin.gnode..."
DEFAULT_REMOTE_PAT= "mearec/mearec_10s.h5"
```

Reply:
Thinking about twice I think having the str as a default argument as you initially proposed is better.